File: Algoritmos Computacionales/verdad.jl
#Calculamos tabla de verdad de (P&Q)v(r&(Sv¬Q))&(¬RvS)
#Pedimos inputs
println("Dame el valor de P")
P = parse(Bool,readline())
println("Dame el valor de Q")
Q = parse(Bool,readline())
println("Dame el valor de R")
R = parse(Bool,readline())
println("Dame el valor de S")
S = parse(Bool,readline())
#Variables auxiliares
A = P && Q
B = (!R) || S
C = S || (!Q)
#Resultados
resultado = A || ((R && C) && B)
println("El valor de la propocición es: ",resultado)

File: Algoritmos Computacionales/verdad.py
#Calculamos tabla de verdad de (P&Q)v(r&(Sv¬Q))&(¬RvS)
#Pedimos inputs
P = bool(input("Dame el valor de P\n"))
Q = bool(input("Dame el valor de Q\n"))
R = bool(input("Dame el valor de R\n"))
S = bool(input("Dame el valor de S\n"))
#Variables auxiliares
A = P and Q
B = (not R) or S
C = S or (not Q)
#Resultados
resultado = A or ((R and C) and B)
print("El valor de la propocición es:", resultado)